### ISSUE TITLE
Table sorting model

### ISSUE THREAD

**Issue by TheWebTech** [Fyrd/caniuse#2437]
We need to add a section for the table sortable and sorted attributes.

http://w3c.github.io/html/tabular-data.html#table-sorting-model

I don't believe any browsers have currently began implementation but doesn't hurt to get it up there, then we can mark when we hear if any of the browsers start implementing in their alpha/beta channels.

**Comment by cvrebert** [comment]
It's been killed and removed from the standard: https://github.com/whatwg/html/pull/526

**Comment by dstorey** [comment]
Note it is still in W3C HTML https://w3c.github.io/html/tabular-data.html#table-sorting-model

**Comment by cvrebert** [comment]
But liable to be removed there too: https://github.com/w3c/html/issues/56

**Comment by Fyrd** [comment]
Yeah, I doubt any browser would still want to pick this up. If that ever changes will re-open.

**Comment by Fyrd** [comment]
Re-opening as reminder for me to add a search result indicating the feature's status.

**Comment by anyotherway** [comment]
sortable (sortable="sortable" in XHTML)
WHATWG & HTML5.1 only. When present, it indicates the table's contents may be sorted by column. It is used in conjunction with the sorted attribute on the th element, which can be set manually or by the user agent when the user clicks a column.

HTML5 Pocket Reference, 5th Edition, O'Reilly, C 2013 - page 135.
Table > Attributes > HTML5 Global Attributes
